edx_sga/__init__.py

```diff
@@ -2,4 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.11.0'
+__version__ = '0.11.0.appsembler.1'
```

Comment:
Let's not put too much dots, since it may break the installation.

```suggestion
__version__ = '0.11.0.appsembler1'
```

Reply:
done